app/routes/payment-holds.js

```diff
@@ -0,0 +1,62 @@
+const { getPaymentHolds, getPaymentHoldCatgories, getPaymentHoldFrns, addPaymentHold, removePaymentHold } = require('../payment-hold')
+const { convertPaymentHolds, convertPaymentHoldCategories, convertPaymentHoldFrns } = require('../payment-hold/utils')
+
+module.exports = [{
+  method: 'GET',
+  path: '/payment-holds',
+  options: {
+    handler: async (request, h) => {
+      const paymentHolds = convertPaymentHolds(await getPaymentHolds(request.query.open ?? false))
```

Comment:
I don't think you need to coalesce this to false as if the parameter is undefined it will be handled by both the existed logic and suggested change above.  If you wanted to make the code clearer to read you could add a default parameter on the call on the `getPaymentHolds` function.

eg
```
const getPaymentHolds = async (open = false) {
...
```